src/web/ui/components/timeline/utils.tsx

```diff
@@ -65,11 +65,12 @@ export const markerStyles = {
   }),
   verticalLine: css({
     position: 'absolute',
-    top: 0,
+    top: '32px',
```

Comment:
If we're using pixels, the unit can be omitted:

```suggestion
    top: 32,
```